query/src/main/kotlin/com/linecorp/kotlinjdsl/query/spec/Froms.kt

```diff
@@ -1,16 +1,16 @@
 package com.linecorp.kotlinjdsl.query.spec
 
 import com.linecorp.kotlinjdsl.query.spec.expression.EntitySpec
-import javax.persistence.criteria.From
+import javax.persistence.criteria.Path
 import javax.persistence.criteria.Root
 
 class Froms internal constructor(
     val root: Root<*>,
-    private val map: Map<EntitySpec<*>, From<*, *>>
+    private val map: Map<EntitySpec<*>, Path<*>>
```

Comment:
-- english
Since the SimpleAssociator returns Paths, Froms has changed to return Path.

I'm wondering if I should change it to Paths later.

-- korean
SimpleAssociator가 Path를 반환하기 때문에 Froms가 Path를 반환하게 바뀐 거군요.

나중에는 Paths라는 이름으로 변경해야 할지에 대해 고민이 되네요.

Reply:
Yes, this can only be converted at the moment.

-- korean
네 이건 현재 변환할 수 밖에 없습니다.